interactions/api/http/request.py

```diff
@@ -161,40 +171,40 @@ async def request(self, route: Route, **kwargs) -> Optional[Any]:
                         self.buckets[route.endpoint] = _bucket
                         # real-time replacement/update/add if needed.
                     if isinstance(data, dict) and (
-                        data.get("errors")
-                        or ((code := data.get("code")) and code != 429 and data.get("message"))
+                        data.get("errors") or (code and code != 429 and message)
                     ):
                         log.debug(
                             f"RETURN {response.status}: {dumps(data, indent=4, sort_keys=True)}"
                         )
                         # This "redundant" debug line is for debug use and tracing back the error codes.
 
-                        raise LibraryException(
-                            message=data["message"], code=data["code"], severity=40, data=data
-                        )
-                    elif isinstance(data, dict) and data.get("code") == 0 and data.get("message"):
+                        raise LibraryException(message=message, code=code, severity=40, data=data)
+                    elif isinstance(data, dict) and code in (0, 403) and message:
```

Comment:
Why is 403 here?

Reply:
I think it's only 0 and 403 always has a proper message? I'm unsure, but it was only 0 before and I'm confused about the change

Reply:
Can we get an update on this?